content/kubernetes/release-notes/6-2-18-releases/_index.md

```diff
@@ -10,6 +10,11 @@ aliases: [
 
 ]
 ---
+{{<warning>}} Due to a change in the SCC, on OpenShift clusters running version 6.2.12 or earlier upgrading to version 6.2.18 or later, where `node:1` is <b>not</b> the master node, the upgrade might get stuck.
+
+For more detail and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster.md#before-upgrading">}}).
```

Comment:
Anchor fix
```suggestion
For more detail and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster#before-upgrading">}}).
```